database/sql/triggers/instead_of_insert_view_coordenador.sql

```diff
@@ -0,0 +1,14 @@
+-- Crie a trigger que é acionada após a inserção na tabela coordenador_administrativo
+CREATE OR REPLACE FUNCTION atualiza_view_coordenador_pedagogico() RETURNS TRIGGER AS $$
+BEGIN
+    -- Atualize a view view_coordenador_pedagogico com as informações inseridas
+    REFRESH MATERIALIZED VIEW view_pessoas_coordenador_pedagogico;
```

Comment:
Essa linha gerou o seguinte erro:

> ERROR:  relation "view_pessoas_coordenador_pedagogico" does not exist
> CONTEXT:  SQL statement "REFRESH MATERIALIZED VIEW view_pessoas_coordenador_pedagogico"

Pelo visto o nome da view está como `view_coordenador_gestor`

```suggestion
    REFRESH MATERIALIZED VIEW view_coordenador_gestor;
```